### DISCUSSION TITLE
Pentesting Extension

### DISCUSSION THREAD

**Discussion by Gaetner84** [skycocker/chromebrew#8018] [Ideas]
First of all I would like to say that this project is simply great, so a big thank you for this first-class work.

I bought a Lenovo Duet 5 and therefore I use your project "crew" to expand crosh accordingly, which works excellently.

What I miss a little in your data and I know that you can't install everything directly are pentesting tools. You already have tools like Nmap in your package sources. But tools like ettercap are still missing.

You could also make your project interesting for pentesters, projects like Kali Linux show that such software can be run almost anywhere. An extension in the pentesting sector would be really great. It might be possible to use the Kali Linux package sources directly.

Thanks again for being able to use crosh productively.

**Comment by satmandu** [comment]
Package contributions and suggestions are welcome!

**Comment by Gaetner84** [comment]
I would definitely use terminal based tools for most of the tools as they run on every system.

I have for a few minutes via:
"crew install wireshark tested"

Unfortunately I don't have a GUI...
Tools such as: ettercap in the ettercap-text-only version.
A list of the tools used in Kali can be found here:

https://www.kali.org/tools/

Unfortunately I can't code myself.

**Reply by Gaetner84** [reply]
sorry here are all tools: https://www.kali.org/tools/all-tools/

**Reply by satmandu** [reply]
Here you go. Let us know if it works for you? https://github.com/chromebrew/chromebrew/pull/8019

**Reply by satmandu** [reply]
![image](https://user-images.githubusercontent.com/1096701/222236761-18d5d23e-4941-4efb-bd76-8a392c22954e.png)

**Reply by Gaetner84** [reply]
Hi, sorry for my late reply, I was at work.

Ettercap runs in the terminal, i.e. the shell and also excellent on the graphical interface, I tested it on the "Lenovo Duet 5" and it works perfectly there!

It's really great, great work!
Now, thanks to such good work, ChromeOS can actually be used productively, making it interesting for pentesters.

Thank you very much for your effort and the great work.
Kind regards Martin

**Comment by uberhacker** [comment]
@Gaetner84: Glad you enjoy Chromebrew.  I try and make it a goal to add at least one new package per week so this is a great list to pull resources.

**Reply by Gaetner84** [reply]
This will be amazing when this is implemented, I'll share it on the Kali pages immediately so the project gets some more exposure. Many use a Chromebook and cannot install another system. With your "CREW" it all becomes child's play and you no longer necessarily need another system.

1000 thanks for this mega project.

**Comment by Gaetner84** [comment]
sorry again I have a question regarding crew. Would it be possible, or would it be possible to integrate the whole thing here at CREW:

https://ubunlog.com/en/katoolin-kali-tools-ubuntu/

With the script it would be possible to install all, or let's say almost all Kali Linux tools, the question is whether you can implement them like this?

Maybe that saves work, it was just an idea.

**Reply by Gaetner84** [reply]
oh ok wie ich sehe ist das Projekt zu Katoolin schon etwas älter: https://github.com/LionSec/katoolin

**Reply by satmandu** [reply]
> sorry again I have a question regarding crew. Would it be possible, or would it be possible to integrate the whole thing here at CREW:
> 
> https://ubunlog.com/en/katoolin-kali-tools-ubuntu/
> 
> With the script it would be possible to install all, or let's say almost all Kali Linux tools, the question is whether you can implement them like this?
> 
> Maybe that saves work, it was just an idea.

That's not going to work, I think. Each software package we have is maintained independently. Sure there are packages which bundle a bunch of software (such as gstreamer), but those use meson or some other setup to compile everything together into a single installation.
